<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/copy.jpg);

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_Noticias_Vg">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#ffffff;&quot;&gt;NOTICIAS DE VIOLENCIA&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Noticias_Nvg">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>351</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#ffffff;&quot;&gt;NOTICIAS DE NO VIOLENCIA&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tbl_Vg">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>401</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tbl_Nvg">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>80</y>
      <width>401</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btt_Reentreno">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>560</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>REENTRENAMIENTO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btt_Guardar_Noticias">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>560</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>GUARDAR NOTICIAS</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>892</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
